### ISSUE TITLE
dmg generation fails on macos catalina

### ISSUE THREAD

**Issue by tajetaje** [fvarrui/JavaPackager#34]
When running mvn packager I get the following error:

pom.xml:

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <properties>
        <maven.compiler.target>11</maven.compiler.target>
        <maven.compiler.source>11</maven.compiler.source>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    </properties>
    <build>
        <sourceDirectory>src</sourceDirectory>
        <resources>
            <resource>
                <directory>src/main/resources</directory>
            </resource>
        </resources>
        <plugins>
            <plugin>
                <groupId>io.github.fvarrui</groupId>
                <artifactId>javapackager</artifactId>
                <version>0.9.7</version>
                <executions>
                    <execution>
                        <phase>package</phase>
                        <goals>
                            <goal>package</goal>
                        </goals>
                        <configuration>
                            <!-- mandatory -->
                            <mainClass>main.java.DebateAppMain</mainClass>
                            <!-- optional -->
                            <bundleJre>true</bundleJre>
                            <generateInstaller>true</generateInstaller>
                            <administratorRequired>false</administratorRequired>
                            <platform>auto</platform>
                            <organizationUrl>https://github.com/tajetaje</organizationUrl>
                            <url>https://github.com/tajetaje/DebateApp</url>
                            <!-- Must provide JRE for macOS: -->
                            <jrePath>${basedir}/../Libraries/Java/jdk-11.0.7-full.jdk</jrePath>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
        </plugins>
    </build>
    <groupId>tajetaje</groupId>
    <artifactId>DebateApp</artifactId>
    <version>2.0.0</version>
    <dependencies>
        <dependency>
            <groupId>org.jfxtras</groupId>
            <artifactId>jmetro</artifactId>
            <version>11.6.12</version>
        </dependency>
        <dependency>
            <groupId>org.controlsfx</groupId>
            <artifactId>controlsfx</artifactId>
            <version>11.0.1</version>
        </dependency>
        <!--        Will put these back in when I implement DOCX support-->
        <!--        <dependency>-->
        <!--            <groupId>org.docx4j</groupId>-->
        <!--            <artifactId>docx4j-core</artifactId>-->
        <!--            <version>11.1.7</version>-->
        <!--        </dependency>-->
        <!--        <dependency>-->
        <!--            <groupId>net.sf.jtidy</groupId>-->
        <!--            <artifactId>jtidy</artifactId>-->
        <!--            <version>r938</version>-->
        <!--        </dependency>-->
    </dependencies>
</project>
```

mvn -e clean packager output:
```
[INFO] --- javapackager:0.9.7:package (default) @ DebateApp ---
[INFO] Packaging app for mac
[info] Removing folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp]
[INFO] License file found: /Volumes/Tag's Drive/Development/DebateApp/LICENSE
[info] Copying resource [/mac/default-icon.icns] to file [/Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp.icns]
[info] Copying stream to file [/Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp.icns]
[INFO] Creating runnable JAR...
[INFO] Building jar: /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp-2.0.0-runnable.jar
[INFO] Creating Mac OS X app bundle...
[INFO] Creating and setting up the bundle directories
[INFO] Copying dependencies to Java folder
[INFO] Copying all dependencies to DebateApp folder ...
[INFO] Copying jmetro-11.6.12.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/jmetro-11.6.12.jar
[INFO] Copying javafx-controls-11.0.2-win.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-controls-11.0.2-win.jar
[INFO] Copying javafx-controls-11.0.2-mac.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-controls-11.0.2-mac.jar
[INFO] Copying javafx-base-11.0.2-win.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-base-11.0.2-win.jar
[INFO] Copying javafx-base-11.0.2-mac.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-base-11.0.2-mac.jar
[INFO] Copying javafx-graphics-11.0.2-win.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-graphics-11.0.2-win.jar
[INFO] Copying javafx-graphics-11.0.2-mac.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-graphics-11.0.2-mac.jar
[INFO] Copying controlsfx-11.0.1.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/controlsfx-11.0.1.jar
[INFO] Copying javafx-base-11.0.2.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-base-11.0.2.jar
[INFO] Copying javafx-web-11.0.2.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-web-11.0.2.jar
[INFO] Copying javafx-web-11.0.2-mac.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-web-11.0.2-mac.jar
[INFO] Copying javafx-graphics-11.0.2.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-graphics-11.0.2.jar
[INFO] Copying javafx-controls-11.0.2.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-controls-11.0.2.jar
[INFO] Copying javafx-swing-11.0.2.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-swing-11.0.2.jar
[INFO] Copying javafx-swing-11.0.2-mac.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-swing-11.0.2-mac.jar
[INFO] Copying javafx-media-11.0.2.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-media-11.0.2.jar
[INFO] Copying javafx-media-11.0.2-mac.jar to /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java/libs/javafx-media-11.0.2-mac.jar
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp-2.0.0-runnable.jar] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources/Java]
[INFO] Bundling JRE ... with /Library/Java/JavaVirtualMachines/liberica-jdk-11-full.jdk/Contents/Home
[INFO] Embedding JRE from /Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk
[info] Copying folder content [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying folder [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/man] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/._man] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/LICENSE] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/._LICENSE] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying folder [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/bin] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/._bin] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying folder [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/include] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/._include] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/release] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/._release] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying folder [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/lib] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/._lib] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/readme.txt] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/._readme.txt] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying folder [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/conf] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/._conf] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying folder [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/jmods] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/../Libraries/Java/jdk-11.0.7-full.jdk/._jmods] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home]
[info] Removing folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/PlugIns/jre/Contents/Home/legal]
[INFO] Creating startup file
[INFO] Copying icon file to Resources folder
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp.icns] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources]
[INFO] Writing the Info.plist file
[INFO] Copying additional resources
[info] Copying file [LICENSE] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app/Contents/Resources]
[info] Executing command: /bin/bash -c codesign --force --deep --sign - /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/DebateApp.app
[INFO] Generating DMG disk image file
[INFO] Creating Applications link
[info] Creating symbolic link [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/Applications] to [/Applications]
[INFO] Copying background image
[info] Copying resource [/mac/background.png] to file [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/.background/background.png]
[info] Copying stream to file [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/.background/background.png]
[INFO] Copying icon file: /Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp.icns
[info] Copying file [/Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp.icns] to folder [/Volumes/Tag's Drive/Development/DebateApp/target/DebateApp/.VolumeIcon.icns]
[INFO] Creating image: /Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp_2.0.0.dmg
[info] Executing command: /bin/bash -c hdiutil create -srcfolder /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp -volname DebateApp -fs HFS+ -format UDRW /Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp_2.0.0.dmg
[error] hdiutil: create failed - No such file or directory
[INFO] Mounting image: /Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp_2.0.0.dmg
[info] Executing command: /bin/bash -c hdiutil attach -readwrite -noverify -noautoopen /Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp_2.0.0.dmg
[error] /bin/bash: -c: line 0: unexpected EOF while looking for matching `''
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  06:24 min
[INFO] Finished at: 2020-06-05T20:45:26-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.github.fvarrui:javapackager:0.9.7:package (default) on project DebateApp: Execution default of goal io.github.fvarrui:javapackager:0.9.7:package failed: No value present -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.github.fvarrui:javapackager:0.9.7:package (default) on project DebateApp: Execution default of goal io.github.fvarrui:javapackager:0.9.7:package failed: No value present
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default of goal io.github.fvarrui:javapackager:0.9.7:package failed: No value present
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:148)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.util.NoSuchElementException: No value present
    at java.util.Optional.get (Optional.java:148)
    at io.github.fvarrui.javapackager.PackageMojo.generateDmgImage (PackageMojo.java:931)
    at io.github.fvarrui.javapackager.PackageMojo.execute (PackageMojo.java:347)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[ERROR] 
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

**Comment by fvarrui** [comment]
Hi again @tajetaje,
I think it's due to whitespaces in volume's name. I'll do some research to be sure and try to fix it.

```
[info] Executing command: /bin/bash -c hdiutil create -srcfolder /Volumes/Tag's Drive/Development/DebateApp/target/DebateApp -volname DebateApp -fs HFS+ -format UDRW /Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp_2.0.0.dmg
[error] hdiutil: create failed - No such file or directory
[INFO] Mounting image: /Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp_2.0.0.dmg
[info] Executing command: /bin/bash -c hdiutil attach -readwrite -noverify -noautoopen /Volumes/Tag's Drive/Development/DebateApp/target/assets/DebateApp_2.0.0.dmg
[error] /bin/bash: -c: line 0: unexpected EOF while looking for matching `''
```

Thanks!

**Comment by tajetaje** [comment]
Yup, that's it. Changed the volume name to TagDrive and the packager worked perfectly, Thanks! And I imagine a quick fix to the issue with the hdiutil is just to enclose the command in "" to ensure there are never any issues.

**Comment by fvarrui** [comment]
This issue has been fixed in **v1.0.0-SNAPSHOT**, which has to be [installed manually](https://github.com/fvarrui/JavaPackager#how-to-build-and-install-the-plugin), until released to Maven Central:
```bash
git clone https://github.com/fvarrui/JavaPackager
cd JavaPackager
mvn install
```
